.idea/misc.xml

```diff
@@ -1,9 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="DesignSurface">
+    <option name="filePathToZoomLevelMap">
+      <map>
+        <entry key="..\:/Users/kriti/StudioProjects/opensource/oppia-android/app/src/main/res/layout-land/topic_practice_subtopic.xml" value="0.21516754850088182" />
+        <entry key="..\:/Users/kriti/StudioProjects/opensource/oppia-android/app/src/main/res/layout-sw600dp-land/topic_practice_subtopic.xml" value="0.16331994645247658" />
+        <entry key="..\:/Users/kriti/StudioProjects/opensource/oppia-android/app/src/main/res/layout-sw600dp-port/topic_practice_subtopic.xml" value="0.190625" />
+        <entry key="..\:/Users/kriti/StudioProjects/opensource/oppia-android/app/src/main/res/layout/topic_practice_subtopic.xml" value="0.1" />
+      </map>
+    </option>
+  </component>
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" project-jdk-name="11" project-jdk-type="JavaSDK">
     <output url="file://$PROJECT_DIR$/build/classes" />
   </component>
   <component name="ProjectType">
     <option name="id" value="Android" />
   </component>
-</project>
+</project>
```

Comment:
We don't need any change to `.idea` file, this must have been done automatically from android studio please do not add this file to your commit. 
Note: Don't delete this file, we need this file we just don't need the changes this file contains.

Reply:
Done.

Reply:
The files are still visible in `Files changed` section.